docs/rpc.md

```diff
@@ -0,0 +1,38 @@
+### `rpc`
+
+Invoke an RPC call to the node in the format:
+`cargo contract rpc [Options] METHOD [PARAMS]`
+
+e.g.
+
+```bash
+cargo contract rpc author_insertKey '"sr25"' '"//ALICE"' \
+      5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY
+
+ cargo contract rpc author_hasKey
+     5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY '"sr25"' \
+```
+ 
+account can be provided as ss58 address:
+`5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY`
+or in hex:
+`0xd43593c715fdd31c61141abd04a99fd6822c8558854ccde39a5684e7a56da27d`
+
+Some of the commands require SS58 address in the string format:
```

Comment:
Is there a way to automatically convert it to string to make it consistent? I see this as a source of user errors in future.

Reply:
> Is there a way to automatically convert it to string to make it consistent? I see this as a source of user errors in future.

You would need to know for which command it is required. In this case, it means hardcoding the command name and matching agains it

Reply:
Sure, then I would create an issue for this, so we can pick it up later